rllib/utils/metrics/tests/test_stats.py

```diff
@@ -429,6 +430,633 @@ def test_basic_throughput():
     )  # default EMA coefficient for throughput is 0.01
 
 
+@pytest.mark.parametrize(
```

Comment:
This test serves two purposes:
1) Make sure we've done the math for every combination of parameters that currently exist on the Stats object.
2) Illustrate the effects that combinations of parameters have. This can help users who want a very distinct behaviour when aggregating metrics to validate their understanding.